docker/cloud.yaml.erb

```diff
@@ -92,7 +92,7 @@ services:
     depends_on:
     - mysql
     volumes:
-    - <%= @config['common']['data_dir'] %>/mysql-backup:/var/backups
+    - <%= @config['common']['backup_dir'] %>/mysql-backup:/var/backups
```

Comment:
I had this in `data_dir` so it would get included in the the backup tar operation, as well as getting replicated with rsync.